manifests/supervisorcluster/1.25/cns-csi.yaml

```diff
@@ -519,6 +519,11 @@ webhooks:
         operations:  ["CREATE", "UPDATE", "DELETE"]
```

Comment:
I think this is also not correct, webhook does not require to Create, Update or Delete persistentvolumeclaims.
Can we confirm if these are really required?

Reply:
ignore this. It is for intercepting create, update and delete requests for persistentvolumeclaims